packages/browser-history/index.js

```diff
@@ -1,7 +1,9 @@
 import createBrowserHistory from 'history/createBrowserHistory';
 import withQuery from 'history-query-enhancer';
-import queryString from 'query-string';
+import { encode, decode } from 'qss';
 
-const history = withQuery(queryString)(createBrowserHistory());
+const history = withQuery({ parse: decode, stringify: encode })(
```

Comment:
This is what the `withQuery` expects, source: https://github.com/yenshih/history-query-enhancer/blob/master/src/withQuery.ts#L12